ui/raidboss/data/07-dt/raid/r4s.ts

```diff
@@ -727,6 +727,25 @@ const triggerSet: TriggerSet<Data> = {
       condition: Conditions.targetIsYou(),
       run: (data) => data.witchgleamSelfCount++,
     },
+    {
+      id: 'R4S Witchgleam Self Reminder',
+      type: 'StartsUsing',
+      netRegex: { id: '95CE', source: 'Wicked Thunder', capture: false },
+      condition: (data) => data.condenserTimer === 'long',
+      delaySeconds: 3,
+      infoText: (data, _matches, output) => {
+        if (data.condenserTimer === 'long') {
+          return output.witchgleamTimes!({ times: data.witchgleamSelfCount - 1 });
+        }
+        return output.witchgleamTimes!({ times: data.witchgleamSelfCount });
+      },
```

Comment:
```suggestion
      infoText: (data, _matches, output) => output.witchgleamTimes!({ times: data.witchgleamSelfCount }),
```

Since there is a `condition` already, e.g. `data.condenserTimer === 'long'`, the fall-through `return` will never fire (and we don't want it to anyway, since short debuff players should not receive an output).

I also don't think we want to subtract one from `data.witchgleamSelfCount`.  I think the way some (if not many) players think about it (and the way that we've written the rest of these triggers), having a long debuff is effectively treated as already having one stack.  (For example, in `R4S Electrope Edge 2 Sidewise Spark`, we say "Spread (3 stacks after)").  So I wouldn't want to be inconsistent here and say "2 stacks (later)" when, in reality, the player will have a 3-stack explosion (and the next trigger will tell them they have 3 stacks after).